controllers/providers/pg/provider.go

```diff
@@ -134,7 +134,14 @@ func (p *Provider) Status(ctx context.Context) (everestv1alpha1.DatabaseClusterS
 
 // Cleanup runs the cleanup routines and returns true if the cleanup is done.
 func (p *Provider) Cleanup(ctx context.Context, database *everestv1alpha1.DatabaseCluster) (bool, error) {
-	return common.HandleDBBackupsCleanup(ctx, p.C, database)
+	done, err := common.HandleDBBackupsCleanup(ctx, p.C, database)
+	if err != nil {
+		return false, err
+	}
+	if !done {
+		return true, nil
```

Comment:
Shouldn't it return `false, nil`? (The comment says `.. returns true if cleanup is done`)

Reply:
Right, it should've, thank you. However the code is not relevant anymore since the `dbb-cleanup` finalizer is deleted.